TaskModel.json

```
@@ -2309,6 +2309,18 @@
 		},
 		{
 			"group": 1,
+			"name": "S1 von %s nach %s",
+			"service": 2,
+			"descriptions": [
+				"Befördere die Fahrgäste pünktlich und sicher durch das Ruhrgebiet.",
+				"Übernehme diese Leistung der S-Bahn Rhein-Ruhr von %s nach %s.",
+				"Der VRR schreibt diese Verbindung zwischen %s und %s aus."
+			],
+			"stations": [ "EDO", "EBO", "EE", "EDG", "KDFF", "KD", "KSO" ],
+			"stopsEverywhere": true,
+			]
```

Die Eckige ist alleine wieder zu viel - außerdem sollte noch dazukommen, welche Kapazitäten überhaupt benötigt werden (da kannst du die schließende eckige Klammer gleich mit verwursten).

```suggestion
			"neededCapacity": [
				{ "name": "passengers", "value": 0 }
			]
```

---

Vielen Dank für die Anmerkungen. Das hat mir sehr weitergeholfen.